lib/live_cluster/manage_controller.py

```diff
@@ -32,7 +32,47 @@
 logger.setLevel(logging.CRITICAL)
 
 
-class ManageLeafCommandController(LiveClusterCommandController):
+class LiveClusterManageCommandController(LiveClusterCommandController):
+    def pre_controller(self, line):
+        self._init_controller_arg()
+
+        if self.controller_arg is None:
+            return
+
+        mod = self.context[-1]
```

Comment:
This only supports 1 mod right? Is there a possibility 2 could be passed in? like "for' and "with"?

Reply:
Food for thought, if modifiers being the second to last argument in context is common maybe you can make a getMods function out of this? Or make context a class? That way it is easier to tell what data is in context and where it goes.

Reply:
This would take some explaining beyond what is reasonable in a chat I think.  To be clear, I did not add this behavior.  I only relocated it as to not pollute other classes inheriting from `CommandController`.  Essentially, this mechanism is allowing required args (controller arg) provided to high-level controllers to be passed down to their children via mods.